File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/C/Exercicio9.c
//9. Faça um programa que calcule e mostre a área de um triângulo. Sabe-se que: Área = (base * altura)/2.

#include <stdio.h>

int main()
{
    float base, altura, area = 0;
    
    printf("Digite a base: ");
        scanf("%f", &base);
    
    printf("Digite a altura: ");
        scanf("%f", &altura);
    
    area = (base * altura) / 2;

    printf("Area do triangulo: %.1f", area);

    return 0;
}

File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/Java/Exercicio9.java
/*9. Faça um programa que calcule e mostre a área de um triângulo. Sabe-se que: Área = (base * altura)/2.*/

import java.util.Scanner;

public class Exercicio9 {
    public static void main(String[] args){

        Scanner scan = new Scanner(System.in);

        System.out.print("Digite o valor da base: ");
        double base = scan.nextDouble();

        System.out.print("Digite o valor da altura: ");
        double altura = scan.nextDouble();

        double area = (base * altura) / 2;

        System.out.printf("%nA área do triângiulo é %.1f.%n", area);

        scan.close();
    }   
}
